lib/tx/AuthTransaction.ts

```diff
@@ -199,7 +199,18 @@ function links2fns(sdk, res, obj, ref) {
     }
 
     var link = obj._links[linkName];
-    
+
+    // replace responses hostname with issuer hostname provided by SDK
+    if (Object.prototype.hasOwnProperty.call(link, "href")) {
+      var matchProtocolDomainHost = /^.*\/\/[^\/]+:?[0-9]?/i;
+      var sdkProvidedissuedDomainHost =link.href.replace(
+        matchProtocolDomainHost,
+        sdk.getIssuerOrigin()
+      );
+      
+      Object.assign(link, { href: sdkProvidedissuedDomainHost });
+    }
+        
```

Comment:
ok this change should actually do it..

Reply:
@aarongranick-okta is this something you can look at? Would love feedback. I would like get a patch in ASAP

Reply:
> @aarongranick-okta is this something you can look at? Would love feedback. I would like get a patch in ASAP
@aakashyadav-okta Is there a way this href is configurable from admin ui? If not,  why this redirect href link is not configurable in okta ui similar to login redirect url?